server/storage/mvcc/watchable_store.go

```diff
@@ -458,7 +452,7 @@ func (s *watchableStore) notify(rev int64, evs []mvccpb.Event) {
 }
 
 func (s *watchableStore) addVictim(victim watcherBatch) {
-	if victim == nil {
+	if victim == nil || len(victim) == 0 {
```

Comment:
```suggestion
	if len(victim) == 0 {
```
Don't think that checking nil is needed here.

Reply:
resolved, thanks